tests_thien/pages/dashboard/dashboard_page.py

```diff
@@ -0,0 +1,73 @@
+from selenpy.element.base_element import BaseElement
+from selenpy.support import browser
+from selenpy.support.conditions import have
+from selenpy.element.text_box import TextBox
+from selenpy.element.combo_box import ComboBox
+from tests_thien.utilities import constant, browser_helper
+from tests_thien.utilities.settings_list import SettingsList
+from tests_thien.utilities.menu_list import MenuList
+
+class DashboardPage():
+    
+    def __init__(self):
+        self._btn_global_setting = BaseElement("class=mn-setting")
+        self._txt_page_name = TextBox("id=name")
+        self._cbb_parent_name = ComboBox("id=parent")
+        self._btn_ok = BaseElement("id=OK")
+        self._lbl_test_module_execution = BaseElement("//div[@title='Test Module Execution']")
+        self._link_page_item= BaseElement("//div[@id='main-menu']//a[text()='%s']")
+        self._item_menu = BaseElement("//a[text()='%s']")
+    
+    def page_title(self,title):
+        return constant.DA_TITLE+ " - " +title
+    
+    def is_page_displayed(self,page_name):
+        browser.wait_until(have.title(self.page_title(page_name)))
+        return browser_helper.get_title() == self.page_title(page_name)
```

Comment:
Why do you need to check here? The first line code `browser.wait_until(have.title(self.page_title(page_name)))` will help you to identify that the page has the expected title or not

Reply:
Sorry. Removed.
I added at the beginning to verify the main page is displayed after logging in
Updated wait to `login()` function in login_page also